WMFComponents/Sources/WMFComponents/Components/Profile/WMFProfileViewModel.swift

```diff
@@ -156,8 +160,20 @@ enum ProfileState {
                                 coordinatorDelegate?.handleProfileAction(.logDonateTap)
                             }
                         )
+//                        ,
+//                        ProfileListItem(
+//                            text: "Year in Review",
+//                            image: .calendar,
+//                            imageColor: WMFColor.blue600,
+//                            hasNotifications: false,
+//                            isDonate: false,
+//                            isLoadingDonateConfigs: false,
+//                            action: {
+//                                coordinatorDelegate?.handleProfileAction(.showYearInReview)
+//                            }
+//                        )
                     ],
-                    subtext: nil
+                    subtext: nil //subtext: "Log in or create an account to get an improved year in review next year"
```

Comment:
You can delete this comment here, I believe this text would belong to the logged out version